client/my-sites/checkout/src/hooks/use-create-payment-complete-callback.tsx

```diff
@@ -237,7 +237,12 @@ export default function useCreatePaymentCompleteCallback( {
 
 			// We need to do a hard redirect if we're redirecting to the stepper.
 			// Since stepper is self-contained, it doesn't load properly if we do a normal history state change
-			if ( isURL( url ) || url.includes( '/setup/' ) ) {
+			// The same is true if we are redirecting to the signup flow, we are restricting it to only 1 specific flow here.
+			if (
+				isURL( url ) ||
+				url.includes( '/setup/' ) ||
+				url.includes( '/start/site-content-collection' )
```

Comment:
`Nonblocking / nitpick:`

Wondering if you could explain a bit more about this specific URL? Not sure I understand what it is. 🤔

Reply:
This is the URL for the content submission form. After checkout, users are redirected to `/start/site-content-collection?siteSlug=<site slug>` to enter content for their BBE site build.